assets/_scss/components/_alerts.scss

```diff
@@ -34,9 +34,12 @@
 }
 
 .usa-alert-heading {
+  @include media($medium-screen) {
+    margin-top: .3rem;
+  }
   margin: {
     bottom: .15em;
```

Comment:
Why is this measurement in em while everything else is in rem? Shouldn't all of them be rem?